components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -49,8 +53,12 @@ export class ProjectsDropdownComponent implements OnInit, OnChanges {
     }
   }
 
-  ngOnChanges(): void {
+  ngOnChanges(changes: SimpleChanges): void {
     this.updateLabel();
```

Comment:
OK, for the moment but want to circle back and put this call gated by `changes.projects`, too.